src/lib/viewers/controls/color-picker/ColorPickerPalette.scss

```diff
@@ -0,0 +1,46 @@
+.bp-ColorPickerPalette {
+    position: absolute;
+    left: -55px;
```

Comment:
I'm a little unsure what we're trying to achieve with this value. If the goal is to center the palette, consider using `left: -50%; transform: translateX(50%);` or some variation.

Reply:
Good callout, it looks like we can go with 
```
right: 50%;
transform: translateX(50%);
```